test/bin/pyutils/generate_common_versions.py

```
@@ -349,12 +349,16 @@ def generate_common_versions(major_version, minor_version):
     # creating PRs that clear the version due to transient failures.
     logging.info("Getting GITOPS_VERSION")
     gitops_version = get_gitops_version(major_version, minor_version)
-    if gitops_version is None:
+    if not gitops_version:
         target_file = pathlib.Path(__file__).resolve().parent / '../common_versions.sh'
         args = ['grep', '-oP', '(?<=GITOPS_VERSION=).*', str(target_file)]
         gitops_version = subprocess.run(args, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, text=True, check=True).stdout.strip()
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)